front/client/router/index.ts

```diff
@@ -4,10 +4,16 @@ function createRouter () {
   return new Router({
     mode: 'history',
     fallback: false,
-    scrollBehavior: () => ({ x: 0, y: 0 }),
+    scrollBehavior: (to) => {
```

Comment:
Whats is this about? Needs comment. Also you can write function properties as `func() {}`